packages/ListBox/src/helpers/applyOnChange.js

```diff
@@ -0,0 +1,49 @@
+import useListBox from "../useListBox";
```

Comment:
Why is it `helpers/applyOnChange.js` when you export a bunch of other helpers? I am not convinced about the folder structure for these `/helpers` or the accompanying `/effects`, `/imperative`.  Also, why is there a `ListBox/store` and a `ListBox/src/store`?

Reply:
`ListBox/store` might be there because was before we move everything to `src` folder maybe I forgot to delete it. 

I'm not seeing it in atom. 🤔
<img width="272" alt="Screen Shot 2019-06-11 at 5 47 47 PM" src="https://user-images.githubusercontent.com/19418590/59315996-1bf8e400-8c71-11e9-8edf-5ff0740d13ca.png">

About folder structure, I'm not sure if this PR should be the one specifically addressing that issue. Is quite complex merging two branches when changing to many folder structures (files and stories, test, depend on it), 

what about if you have a folder structure that you want to see you can add it here, and I can address it in another PR?

Reply:
Sounds reasonable. After you merge please make sure to follow up on this with me.